packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -724,6 +724,43 @@ function updateTsLintConfig(): Rule {
   };
 }
 
+function updateRootTsConfig(): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    const tsConfigPath = '/tsconfig.json';
+    const buffer = host.read(tsConfigPath);
+    if (!buffer) {
+      return;
+    }
+
+    const tsCfgAst = parseJsonAst(buffer.toString(), JsonParseMode.Loose);
+    if (tsCfgAst.kind != 'object') {
+      throw new SchematicsException('Invalid tsconfig. Was expecting an object');
```

Comment:
The message should indicate the root tsconfig is the problematic one.  Also, I'm not sure that this would be fatal.  Maybe log an error/warning and return?  Same for the check below.

Reply:
I think it is fatal since other schematics  such as generate library always extend the root ts config.

What do you think?

Reply:
True.  Just the verbiage adjustment then.